content/ripplerest_api.md

```diff
@@ -495,10 +496,29 @@ An order object describes an offer to exchange two currencies
 | type  | String (`buy` or `sell`) | Whether the order is to buy or sell. |
 | taker_pays | String ([Amount Object](#amount_object)) | The amount the taker must pay to consume this order. |
 | taker_gets | String ([Amount Object](#amount_object)) | The amount the taker will get once the order is consumed. |
+| sequence   | Number | The sequence number of the transaction that created the order. Used in combination with account to uniquely identify the order. |
```

Comment:
These are added in this proposed change: https://github.com/ripple/ripple-rest/pull/262.